native/src/seal/encryptor.cpp

```diff
@@ -88,6 +88,21 @@ namespace seal
     void Encryptor::encrypt_zero_internal(
         parms_id_type parms_id, bool is_asymmetric, bool save_seed, Ciphertext &destination,
         MemoryPoolHandle pool) const
+    {
+        // Do not export noise.
+        StaticPolynomialArray u_destination;
```

Comment:
What does the word static mean in this context? Usually it means global. Maybe call this RingPolynomial?

Reply:
I suppose static and global are not equivalent in my mind, but I chose the name to be a foil to the "DynArray" class which allows for updating the size of the array. This is similar construct but once the size is set it can't be changed. 

I am probably going to add that a memory slot can only be written once. 

But also RingPolynonial (or just PolynomialArray) also works. I don't have a strong opinion.